jobs/keepalived/templates/bin/action.sh.erb

```diff
@@ -41,10 +41,10 @@ if [[ "$iaas_name" == "aws" ]]; then
   if [[ "$STATE" == "MASTER" ]]; then
     network_metadata=$("$AWS_CLI_CMD" ec2 describe-network-interfaces --filter "Name=private-ip-address,Values=${SOURCE_IP}" | "${JQ_CMD}" -r --arg SUBNET_ID "$SUBNET_ID" '.[][] | select(.SubnetId==$SUBNET_ID)')
     export ENI_ID=$(echo "$network_metadata" | "${JQ_CMD}" -r '.NetworkInterfaceId')
-    $AWS_CLI_CMD ec2 assign-private-ip-addresses --network-interface-id $ENI_ID --allow-reassignment --private-ip-addresses $virtual_ip
-    attach_vip_to_interface
+    $AWS_CLI_CMD ec2 assign-private-ip-addresses --network-interface-id $ENI_ID --allow-reassignment --private-ip-addresses $virtual_ip >> /var/vcap/sys/log/keepalived/keepalived.log 2>&1
```

Comment:
lets place the location as a variable in maybe the job_properties file? Since, it has been used here in numerous places, if the location changes by chance, will need several corrections.